mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -54,6 +55,7 @@
     service: extract_api_info_for_service(service, _REST_API_PATH_PREFIX)
     for service in [MlflowService, DatabricksMlflowArtifactsService]
 }
+_MAX_CREDENTIALS_REQUEST_SIZE = 2000  # Max number of artifact paths in a single credentials request
```

Comment:
Leave a blank line between L57 and L58?

Reply:
Good idea - the positioning of this definition is a bit odd. I've moved this up above `_SERVICE_AND_METHOD_TO_INFO` and below `_DOWNLOAD_CHUNK_SIZE` so that it's next to other integer constants.